qabelbox/src/test/java/de/qabel/qabelbox/storage/notifications/StorageNotificationTest.java

```diff
@@ -0,0 +1,110 @@
+package de.qabel.qabelbox.storage.notifications;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.content.Context;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.annotation.Config;
+import org.robolectric.shadows.ShadowNotification;
+import org.robolectric.shadows.ShadowNotificationManager;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+import de.qabel.qabelbox.BuildConfig;
+import de.qabel.qabelbox.R;
+import de.qabel.qabelbox.SimpleApplication;
+import de.qabel.qabelbox.storage.BoxUploadingFile;
+
+import static org.junit.Assert.assertEquals;
+import static org.robolectric.Shadows.shadowOf;
+
+@RunWith(RobolectricGradleTestRunner.class)
+@Config(application = SimpleApplication.class, constants = BuildConfig.class)
+public class StorageNotificationTest {
+
+    private static final String TEST_OWNER = "owner";
+    private static final String UPLOAD_PATH = "path";
+
+    private NotificationManager notificationManager;
+    private StorageNotificationManager storageNotificationManager;
+
+    @Before
+    public void setUp() {
+        notificationManager = (NotificationManager) RuntimeEnvironment.application
+                .getSystemService(Context.NOTIFICATION_SERVICE);
+        shadowOf(notificationManager).cancelAll();
```

Comment:
I don't think that is needed, as the environment is recreated for each test afaik.